FATO/.htaccess

```diff
@@ -1,26 +1,26 @@
+Options +FollowSymLinks
 # Turn off MultiViews
-Options +FollowSymLinks -MultiViews
+Options -MultiViews
 
-# Directive to ensure *.rdf *.owl files served as appropriate content type,
+# Directive to ensure *.rdf files served as appropriate content type,
 # if not present in main apache config
-AddType application/rdf+xml .rdf .owl
+AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
+AddType text/turtle .ttl
 
-# Rewrite engine setup
-RewriteEngine On
+RewriteEngine on
 
-# Rewrite rule to serve HTML content from the vocabulary URI if requested
-RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml)
+#Rewrite rules for FATO
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml|text/\*|\*/\*)
 RewriteCond %{HTTP_ACCEPT} text/html [OR]
 RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_ACCEPT} text/\* [OR]
+RewriteCond %{HTTP_ACCEPT} \*/\* [OR]
 RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
-RewriteRule ^$ https://gmparg.github.io/FATO/ [R=303,L]
-
-# Rewrite rule to serve RDF/XML content from the vocabulary URI if requested
-RewriteCond %{HTTP_ACCEPT} application/rdf\+xml [OR]
-RewriteCond %{HTTP_ACCEPT} application/owl\+xml
-RewriteRule ^$ https://gmparg.github.io/FATO/ontologies/FATO.owl [R=303,L]
-
-# Choose the default response
-# ---------------------------
-# Default Rule
-RewriteRule ^$ https://gmparg.github.io/FATO/ [R=303,L]
+RewriteRule ^$ https://gmparg.github.io/FATO/  [R=303,NE,L]
+RewriteCond %{HTTP_ACCEPT} ^.*application/rdf\+xml.* 
+RewriteRule ^asteroids$ https://gmparg.github.io/FATO/ontologies/FATO.owl [R=303,NE,L]
```

Comment:
asteroids?  is that what you want or did this get copied from somewhere else?

Reply:
Thanks so much for noticing - I forgot to remove it when I copied and pasted from the example! I'll close this and create a new one